Bump version to v4.0.0 and add CONTRIBUTING.md (#29)
poolvos/django-nacl-fields: CONTRIBUTING.md

# Contributing

When contributing to this repository, please first discuss the change you wish
to make via issue, email, or any other method with the owners of this repository
before making a change. 

Please note we have a code of conduct, please follow it in all your interactions
with the project.

## Pull Request Process

1. On a new release, increase the version numbers in
`nacl_encrypted_fields/__init__.py` to the new version that this Pull Request
would represent. The versioning scheme we use is [SemVer](http://semver.org/).

> 2. You may merge the Pull Request in once you have the sign-off of al lease on

Reviewer: ```suggestion
2. You may merge the Pull Request in once you have the sign-off of at least one
```
